torch/_torch_docs.py

```diff
@@ -3283,6 +3283,46 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.igamma,
+           r"""
+igamma(input, other, *, out=None) -> Tensor
+
+Computes the regularized lower incomplete gamma function:
+
+.. math::
+    \text{out}_{i} = \frac{1}{\Gamma(\text{input}_i)} \int_0^{\text{other}_i} t^{\text{input}_i-1} e^{-t} dt
+
+where both :math:`\text{input}_i` and :math:`\text{other}_i` are weakly positive
+and at least one is strictly positive.
+If both are zero or either is negative then :math:`\text{out}_i=\text{nan}`.
+:math:`\Gamma(\cdot)` in the equation above is the gamma function,
+
+.. math::
+    \Gamma(\text{input}_i) = \int_0^\infty t^{\text{input}_i-1} e^{-t} dt.
+
+See :func:`torch.lgamma` for a related function.
+Supports :ref:`broadcasting to a common shape <broadcasting-semantics>`
```

Comment:
nit: linebreak here so the "Supports..." statement is its own paragraph.

Reply:
This is also done